test/test-pages/001/expected-metadata.json

```diff
@@ -0,0 +1,5 @@
+{
+  "title": "Get your Frontend JavaScript Code Covered | Code",
+  "byline": "Nicolas Perriault —",
+  "excerpt": "width=device-width"
```

Comment:
Yeah, this exposes some real issues :/ I've seen some similar problems in the reading list UI on Fennec. Well we can start by trying to fix these testcases!